.proxyrc

```diff
@@ -1,6 +1,6 @@
 {
   "/v1": {
-    "target": "http://localhost:9001/",
+    "target": "http://localhost:9000/",
```

Comment:
I must admit I'm quite confused about the 9000 vs 9001 - the manifests/helm chart only exports 9001, so what's listening to 9000?

Reply:
[The `](https://github.com/weaveworks/weave-gitops/blob/c11faf222d0c648c16589fb01477d841c5a2d0e8/Tiltfile#L29)